thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/bao/jdbc/MergedAnomalyResultManagerImpl.java

```diff
@@ -257,6 +257,16 @@ public MergedAnomalyResultDTO findById(Long id) {
     return convertMergedAnomalyBean2DTO(list);
   }
 
+  @Override
+  public List<MergedAnomalyResultDTO> findByStartTimeInRangeAndDetectionConfigId(long startTime, long endTime,
+      long detectionConfigId) {
+    Predicate predicate =
+        Predicate.AND(Predicate.LT("startTime", endTime), Predicate.GT("endTime", startTime),
```

Comment:
Normally the convention is left inclusive and right exclusive.

Reply:
this method is to get the anomalies in the time range: using the 

> anomalyStart < endtime and anomalyEnd > startTime

. The logic is the same as findByStartTimeInRangeAndFunctionId.